tools/sdk/ld/Makefile

```diff
@@ -0,0 +1,17 @@
+
+CPP = ../../../tools/xtensa-lx106-elf/bin/xtensa-lx106-elf-cpp
+COMMON = eagle.app.v6.common.ld
+WARN = "\/\* warning: AUTOGENERATED from $(COMMON)\*.h \*\/"
+
+all:
+	@echo "available rules:"
+	@echo "  iram: move c++ vtables to iram (= default in git master repository)"
+	@echo "  heap: move c++ vtables to heap"
+
+heap:
+	@(echo "$(WARN)"; $(CPP) -E -DVTABLES_IN_DRAM $(COMMON).h | grep -v '^#') > $(COMMON)
+	@echo "c++ vtables moved to heap"
+
+iram:
+	@(echo "$(WARN)"; $(CPP) -E -DVTABLES_IN_IRAM $(COMMON).h | grep -v '^#') > $(COMMON)
+	@echo "c++ vtables moved to iram"
```

Comment:
-CC to preserve comments

Can you add a "flash:" branch, with the appropriate `#if defined(VTABLES_IN_FLASH)` to the makefile and app.ld file as well?